test/test_static_robot_wrapper.py

```diff
@@ -0,0 +1,373 @@
+        equirect_sensor_spec.uuid = "equirect_rgba_sensor"
+        equirect_sensor_spec.sensor_type = habitat_sim.SensorType.COLOR
+        equirect_sensor_spec.resolution = [
+            settings["height"],
+            settings["width"],
+        ]
+        equirect_sensor_spec.position = [0, settings["sensor_height"], 0]
+        for k in kw_args:
+            setattr(equirect_sensor_spec, k, kw_args[k])
+        return equirect_sensor_spec
+
+    if settings["equirect_rgba_sensor"]:
+        equirect_rgba_sensor_spec = create_equirect_spec(
+            uuid="equirect_rgba_sensor"
+        )
+        sensor_specs.append(equirect_rgba_sensor_spec)
+
+    if settings["equirect_depth_sensor"]:
+        equirect_depth_sensor_spec = create_equirect_spec(
+            uuid="equirect_depth_sensor",
+            sensor_type=habitat_sim.SensorType.DEPTH,
+            channels=1,
+        )
+        sensor_specs.append(equirect_depth_sensor_spec)
+
+    if settings["equirect_semantic_sensor"]:
+        equirect_semantic_sensor_spec = create_equirect_spec(
+            uuid="equirect_semantic_sensor",
+            sensor_type=habitat_sim.SensorType.SEMANTIC,
+            channels=1,
+        )
+        sensor_specs.append(equirect_semantic_sensor_spec)
+
+    # create agent specifications
+    agent_cfg = habitat_sim.agent.AgentConfiguration()
+    agent_cfg.sensor_specifications = sensor_specs
+    agent_cfg.action_space = {
+        "move_forward": habitat_sim.agent.ActionSpec(
+            "move_forward", habitat_sim.agent.ActuationSpec(amount=0.25)
+        ),
+        "turn_left": habitat_sim.agent.ActionSpec(
+            "turn_left", habitat_sim.agent.ActuationSpec(amount=10.0)
+        ),
+        "turn_right": habitat_sim.agent.ActionSpec(
+            "turn_right", habitat_sim.agent.ActuationSpec(amount=10.0)
+        ),
+    }
+
+    # override action space to no-op to test physics
+    if sim_cfg.enable_physics:
+        agent_cfg.action_space = {
+            "move_forward": habitat_sim.agent.ActionSpec(
+                "move_forward", habitat_sim.agent.ActuationSpec(amount=0.0)
+            )
+        }
+
+    return habitat_sim.Configuration(sim_cfg, [agent_cfg])
+
+
+def simulate(sim, dt, get_observations=False):
+    r"""Runs physics simulation at 60FPS for a given duration (dt) optionally collecting and returning sensor observations."""
+    observations = []
+    target_time = sim.get_world_time() + dt
+    while sim.get_world_time() < target_time:
+        sim.step_physics(1.0 / 60.0)
+        if get_observations:
+            observations.append(sim.get_sensor_observations())
+    return observations
+
+
+@pytest.mark.skipif(
+    not osp.exists("data/robots/franka_robot"),
+    reason="Test requires Franka robot URDF and assets.",
+)
+@pytest.mark.skipif(
+    not habitat_sim.built_with_bullet,
+    reason="Robot wrapper API requires Bullet physics.",
+)
+
+def test_franka_robot_wrapper():
```

Comment:
To reduce duplication of the boiler plate code setting up and making configurations, let's move this function into test_robot_wrapper.py.

Reply:
done!